### PR TITLE
Fixes: Use file naming conventions, Hero section responsive issue, and Remove redundant code segments

### PR DESCRIPTION
**Pull request by LayanJay**
- add a NextImage component
- move partners logos into lib/logoData.ts file
- render images using a map()
- use file naming conventions in modules
- fixed hero section responsive issue

### CODE REVIEW

File: components/NextImage.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Image from 'next/image'
+
+interface ImageProps {
+    src: StaticImageData
+    alt: string
+    quality?: number
+    layout?: 'fixed' | 'responsive' | 'intrinsic' | 'fill' | undefined
+    width?: number | undefined
+    height?: number | undefined
+    placeholder?: 'blur' | 'empty' | undefined
```
Comment (chamodshehanka) [inline components/NextImage.tsx]: if these props are optional why did you put undefined type

File: pages/index.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-import HeroSection from "../modules/HomePage/hero";
-import MakeGoodThings from "../modules/HomePage/makeGoodThings";
-import WhatWeDo from "../modules/HomePage/WhatWeDo";
-import OurPartners from "../modules/HomePage/ourPartners";
-import Footer from "../components/Footer";
+import HeroSection from '../modules/HomePage/HeroSection'
+import MakeGoodThings from '../modules/HomePage/MakeGoodThings'
+import WhatWeDo from '../modules/HomePage/WhatWeDo'
+import OurPartners from '../modules/HomePage/OurPartners'
+import Footer from '../components/Footer'
```
Comment (chamodshehanka) [inline pages/index.tsx:9]: @sithumonline @LayanJay can you guys stick to one comma pattern

**Review by chamodshehanka — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 2 above)
Comment (LayanJay) [reply pages/index.tsx:9]: in prettierrc files, it defined as single-quote true

Comment (chamodshehanka) [reply pages/index.tsx:9]: Ohh! okay then

**Review by chamodshehanka — APPROVED** [review]

File: lib/logoData.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+// partners logos
+import foss_dark from '../public/logos/foss_dark.svg'
+import Cs2 from '../public/logos/CS2.png'
+import fcsc from '../public/logos/fcsc.png'
```
Comment (sithumonline) [inline lib/logoData.ts:4]: @LayanJay logoData -> LogoData

**Pull request merged**